frontend/src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import NavBar from './components/navbar'
 
 
 function App() {
   return (
     <div className="App">
+      <NavBar />
       <header className="App-header">
```

Comment:
I think its time for us to remove the items in the header tag and just leave the navbar component in there. If we do not remove the all of the contents in the header tag, we get two things being shown at once. Try it out by running the application and navigating to another page.

Reply:
Done.